turso/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "turso-example"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+axum = { version = "0.6.10", features = ["macros"] }
```

Comment:
We should be able to bump this to 0.6.18, and I think the macros feature is for the debug_handler, right?